CHANGES

```diff
@@ -1,5 +1,19 @@
 Version 2.0
 ===========
+Backwards Incompatible Changes
+------------------------------
+* Any installed ZenPacks using older versions of zenpacklib.py will continue to function unchanged.
+* Using version 2.0 is slightly different.  The __init__.py file import statements should now contain the following:
+
+.. code-block:: python
+
+	from ZenPacks.zenoss.ZenPackLib import zenpacklib
+	CFG = zenpacklib.load_yaml()
+	schema = CFG.zenpack_module.schema
```

Comment:
there's nothing that exports it to the namespace i guess -- want to take a look?